docs/r2dbc.md

```diff
@@ -157,20 +157,23 @@ username.
 * Postgres: If the IAM email ends with `.gserviceaccount.com`, remove
   the `.gserviceaccount.com` suffix from the email.
 
-For example, if the full IAM user account
-is `my-sa@my-project.iam.gserviceaccount.com`,
-You would update the code, replacing these examples with the appropriate values
-for your database engine:
-
-|               | Mysql | Postgres             |
-|---------------|-------|----------------------|
-| <PROTOCOL>    | mysql | postgresql           |  
-| <IAM_DB_USER> | my-sa | my-sa@my-project.iam |
+For example, if the full IAM user account is 
+`my-sa@my-project.iam.gserviceaccount.com`, then the shortened database username
+would be `my-sa` for Mysql, and `my-sa@my-project.iam` for Postgres. 
 
 **Note:** a non-empty string value for the `password` property must be set.
 While this property will be ignored when connecting with the Cloud SQL Connector
 using IAM auth, leaving it empty will cause driver-level validations to fail.
 
+#### Example
+
+Replace these parameters in the example based on your database type:
+
+|          | PROTOCOL   | IAM_DB_USER          |
+|----------|------------|----------------------|
+| Mysql    | mysql      | my-sa                |  
```

Comment:
Do we need this first column?

Reply:
It's not strictly required to interpret the table, but I think it improves the clarity of the documentation.